### ISSUE TITLE
helm-mini: Show circe channels in buffer list

### ISSUE THREAD

**Issue by fleimgruber** [jorgenschaefer/circe#305]
Neither `helm-mini` nor `helm-buffers-list` show circe channels that have been auto-joined but not yet visited. The channel buffers appear in the list of completions if they have been visited at least once, though.  

So the actual question: Is there a way to add auto-joined circe channels to helm completion lists?

**Comment by wasamasa** [comment]
Does `(buffer-list)` (try evaluating it in `M-x ielm` or `M-:`) show them?  If not, there isn't much that can be done about that.

**Comment by fleimgruber** [comment]
Yes, `(buffer-list)` shows both the "auto-joined" ones and the "auto-joined-and-visited-at-least-once" ones.

**Comment by wasamasa** [comment]
I've upgraded helm and tried `(circe "Freenode" :nick "not_wasamasa" :channels '("#orbment" "#rackmonkey"))` which from my understanding should auto-join those two channels (I'm otherwise using ZNC which somehow does the right thing).  While I don't initially get those two channels in `helm-mini` (while they are immediately available in `M-x list-buffers`), they do eventually turn up there after waiting a bit. Weird, but not unimaginable for helm, considering I handed in a way more bizarre bug for it.

**Comment by fleimgruber** [comment]
Thanks for trying to reproduce this.  "waiting a bit" has been going on for one hour by now and still no jazz.  I will try this again after the next helm upgrade.
